dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/message/StreamingDecoder.java

```diff
@@ -37,13 +37,6 @@ interface FragmentListener {
          */
         void onFragmentMessage(InputStream rawMessage);
 
-        /**
-         * @param rawMessage raw message
-         */
-        default void onFragmentMessage(InputStream dataHeader, InputStream rawMessage) {
-            onFragmentMessage(rawMessage);
-        }
-
         default void onClose() {}
```

Comment:
**Issue**: Removing the overloaded method onFragmentMessage from FragmentListener interface might lead to backward compatibility issues with existing implementations relying on this method. <br> **Fix**: Reinstate the removed method to maintain backward compatibility or provide an alternative solution for transition. <br> **Code Suggestion**: 
 ```
 @@ -37,5 +37,9 @@
  +        /**
  +         * @param rawMessage raw message
  +         */
  +        default void onFragmentMessage(InputStream dataHeader, InputStream rawMessage) {
  +            onFragmentMessage(rawMessage);
  +        }
 ```